### ISSUE TITLE
is kfctl dead?

### ISSUE THREAD

**Issue by speaknowpotato** [kubeflow/kfctl#501]
it seems that kfctl is dead, 
1. kfctl channel in Kubeflow slack has been archived. 
2. no more active commits since Mar 2021. 

shall we go to https://github.com/kubeflow/manifests to install kubeflow instead?
is there any official announcement from the Kubeflow community? 
Does anyone have thoughts?

**Comment by jbottum** [comment]
@speaknowpotato thanks for your question.   As far as I know, no one in the Community is developing or supporting kfctl.   Some systems and users still use it but there isn't any support.   

The manifest docs do say...."Starting from Kubeflow 1.3, all components should be deployable using kustomize only."   but maybe we need to be more explicit about kfctl and update the kfctl repo. 



We discussed this over several releases 1.2, 1.3 and 1.4.     Is your concern that folks will use kfctl because we are not explicit on the kfctl repo ?

@kubeflow/wg-manifests-leads and @shannonbradshaw Do you have any comments or proposed fixes here?

/area docs
/kind question
/priority p2

**Comment by speaknowpotato** [comment]
@jbottum thanks for your quick response.
yeah, that's my concern, 
it's a little bit confusing for me at first, maybe other people who are new to Kubeflow as well, 
cause there isn't any update on this repo talking about that kfctl is dead.

**Comment by 64J0** [comment]
Hello people, so, what I understood is that this project should not be used anymore, right? 

The instructions for installation in Azure are not updated then, since it points to this tool.

* https://www.kubeflow.org/docs/distributions/azure/deploy/install-kubeflow/

**Comment by thesuperzapper** [comment]
@64J0 @speaknowpotato you are correct that `kfctl` is no longer a supported method of installing Kubeflow.

However, I imagine that `deployKF` will meet your needs, and is significantly more powerful than `kfctl`!

## deployKF Repo: https://github.com/deployKF/deployKF

**Comment by thesuperzapper** [comment]
@zijianjoy can we please archive the `kubeflow/kfctl` repo?

**Comment by thesuperzapper** [comment]
Alternatively, @james-jwu and @chensun should have permission to archive this repo.

**Comment by 64J0** [comment]
Great, thanks for sharing @thesuperzapper. 

I agree that this repository must be archived to avoid mistakes in the future.
